### PR TITLE
Config & misc cleanup

### PR DESCRIPTION
**Pull request by mattwb65**

### CODE REVIEW

File: src/main/java/factionsplusplus/FactionsPlusPlus.java [hunk 1]
```diff
@@ -39,12 +39,12 @@ public class FactionsPlusPlus extends JavaPlugin {
     private final String pluginVersion = "v" + getDescription().getVersion();
     @Inject private ActionBarService actionBarService;
     @Inject private ConfigService configService;
-    @Inject private DataService dataService;
-    @Inject private Scheduler scheduler;
-    @Inject private CommandService commandService;
+    @Inject private Provider<DataService> dataService;
```
Comment (nortoh) [inline src/main/java/factionsplusplus/FactionsPlusPlus.java:42]: Is there an import needed for this? Just wondering

File: src/main/java/factionsplusplus/commands/BaseCommand.java [hunk 2]
```diff
@@ -247,15 +256,25 @@ public void listCommand(CommandContext context) {
...
+                .stream()
+                .map(base -> {
+                    final boolean isOwnBase = base.getFaction().equals(context.getExecutorsFaction().getUUID());
+                    final String baseName = (isOwnBase ? base.getName() : String.format(
+                        "%s (%s)",
+                        base.getName(),
+                        this.dataService.getFaction(base.getFaction()).getName()
+                    ));
+                    return String.format("<color:yellow><lang:BaseList.Base:'<color:white>%s'>", baseName);
+                }).collect(Collectors.joining("<newline>"))
```
Comment (nortoh) [inline src/main/java/factionsplusplus/commands/BaseCommand.java:276]: This is fire stuff

File: src/main/java/factionsplusplus/FactionsPlusPlus.java (hunk 1 above)
Comment (mattwb65) [reply src/main/java/factionsplusplus/FactionsPlusPlus.java:42]: yeah, it's there already: import javax.inject.Provider;

File: src/main/java/factionsplusplus/commands/BaseCommand.java (hunk 2 above)
Comment (mattwb65) [reply src/main/java/factionsplusplus/commands/BaseCommand.java:276]: yeah minimessage is dope

**Pull request merged**